jenkins/opensearch/distribution-build.jenkinsfile

```diff
@@ -594,6 +618,20 @@ pipeline {
         }
     }
     post {
+        always {
+            node(AGENT_X64) {
+                script {
+                    buildInfoYaml(
+                        componentName: COMPONENT_NAME,
+                        status: currentBuild.result,
+                        stage: "COMPLETE"
```

Comment:
Using `Finalize_stage` better than `COMPLETE` here, not to be confused with `STARTED/NOT_STARTED`.